Assets/Src/Scripts/Input/WebCamSource.cs

```diff
@@ -0,0 +1,158 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Diagnostics;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using Debug = UnityEngine.Debug;
+
+public class WebCamSource : MonoBehaviour {
+  public Dropdown CameraDropdown;
+  public Dropdown ResolutionDropdown;
+  public Dropdown FpsDropdown;
+  public Dropdown FlipDropdown;
+  public string DeviceName;
+  public int Fps = 30;
+  public bool Flip = false;
+  [SerializeField] private RenderTexture _buffer;
+  private WebCamTexture _webCam;
+  private int _width = 1280;
+  private int _height = 720;
+  private List<string> _devicesName = new List<string>();
+  private List<string> _resolutionOpt = new List<string>(){"1280X720","1600X900","1920X1080"};
+  private List<string> _fpsOpt = new List<string>(){"30Hz","60Hz"};
+  private List<string> _flipOpt = new List<string>(){"默认","水平翻转"};
+  //Start button event
+  public void Play() {
+    if(_webCam == null) {
```

Comment:
```suggestion
    if (_webCam == null) {
```

There are a lot of empty space issues like this one. Please fix them based on the example code of the Google C# coding style. Thanks,